### DISCUSSION TITLE
Esc to turn off filters

### DISCUSSION THREAD

**Discussion by cmod** [plausible/analytics#575] [Anything Else]
Loving Plausible so far. 

One little feature that I thought would be helpful — being able to press Esc to turn off filters / revert to the standard view. 

A small thing but would make drilling down into sub-views easier to pop into and out of. 

Cheers.

**Comment by metmarkosaric** [comment]
could be nice for sure! thanks for the idea @cmod!

**Comment by cmod** [comment]
You guys implemented this! Thank you! ❤️❤️

**Reply by metmarkosaric** [reply]
Thank you! ❤️ Here's the full list of shortcuts at the moment https://plausible.io/docs/keyboard-shortcuts
